Core/Src/fault.c

```diff
@@ -1,3 +1,45 @@
 #include "fault.h"
 
 osMessageQueueId_t fault_handle_queue;
+
+//Function to determine priority base on severity
+void priority(void) {
```

Comment:
So this should be the main "task" which we will assign a priority and tell the kernel to schedule on boot. For references on other tasks, check the functions that start with `vSomething` in monitor.c. These have a thread attribute struct associated with them that we use to initialize in main.c